Anton.Konovalov/task1.rb

```diff
@@ -0,0 +1,24 @@
+# Task 1.1. Fill with random numbers from -10 to 10 two-dimensional array of 3x3 elements
+array = Array.new(3) { Array.new(3) { rand -10..10 } }
+p array
+
+# Task 1.2. Calculate the sum of all positive elements
+p array.flatten.find_all{|i| i>0}.inject{|i,sum| i+sum}
```

Comment:
I'd recommend introducing a new variable that will contain an array with only positive elements. 
```suggestion:-0+1
array_of_possitive_numbers = array.flatten.find_all(&:positive?)
array_of_possitive_numbers.inject{ |i, sum| i + sum } // or use #sum
```

So you can reuse array_of_possitive_numbers later in code.